scripts/commands.js

```diff
@@ -35,12 +37,17 @@ program
 
 program
   .command('create_dist')
-  .option('--mac_signing_identifier', 'The identifier to use for signing')
+  .option('-C <build_dir>', 'build config (out/Debug, out/Release')
   .option('--target_arch <target_arch>', 'target architecture', 'x64')
+  .option('--mac_signing_identifier <id>', 'The identifier to use for signing')
+  .option('--mac_signing_keychain <keychain>', 'The identifier to use for signing', 'login')
   .option('--debug_build <debug_build>', 'keep debugging symbols')
   .option('--official_build <official_build>', 'force official build settings')
+  .option('--brave_google_api_key <brave_google_api_key>')
```

Comment:
these should only be needed at build time and not packaging

Reply:
Currently create_dist will run the build if it hasn't already

Reply:
Yeah should still be able to do that, won't rebuild if you already ran `yarn run build`. But think it might make sense to expose the target in a more generic way so we don't have to make sure all the options stay in sync.

Reply:
I think what might make sense here is to remove the call to `util.run('gn', ['gen', config.outputDir, '--args="' + args + '"'], cmdOptions)` in `createDist` if no options are specified. Then running `npm run create_dist` will use the existing args.gn instead of overwriting it